features/steps/example_component.go

```diff
@@ -13,11 +13,11 @@ import (
 	kafka "github.com/ONSdigital/dp-kafka/v2"
 	"github.com/ONSdigital/dp-kafka/v2/kafkatest"
 	dphttp "github.com/ONSdigital/dp-net/http"
+	"github.com/pkg/errors"
```

Comment:
This package is essentially deprecated now, as of Go 1.13 all the major usages have been integrated into the standard 'errors' package.
Instead of using errors.Wrap you can use `fmt.Errorf("message string: %w", err)` using the %w verb to wrap the error

Reply:
Removed all dependencies with this package.